### PR TITLE
Feature - 6039 - intro to resume review

### LINKED ISSUE #6039: ✨ Application Revamp: Review your Resume intro page
## ✨ Feature

Intro page for the Review your Resume step.

## 🕵️ Details

Just text, with links to the actual resume step, and back to the Applicant Dashboard

## 🎨 Design File

https://www.figma.com/file/BXYVsu1fo9LS2IySJDUINY/Applicants---Job-application?node-id=636-59839&t=uYeh8QDrmOQsNpJQ-0

## 📸 Screenshot

![image](https://user-images.githubusercontent.com/4313717/227045346-56553b8d-0ced-4314-b5f6-805faef9a4a5.png)

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

| EN                                                                                                                                                                                                                    | FR                                                                                                                                                                                                                                                                               | Desc                                                                                |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
|     Update and review your résumé information.                                                                                                                                                                        |     Mettez à jour et   révisez les renseignements de votre CV.                                                                                                                                                                                                                   |     Application step to begin working on profile, page subtitle                     |
|     Great work! On to your résumé.                                                                                                                                                                                    |     Excellent travail!   Passons à votre CV.                                                                                                                                                                                                                                     |     Application step to begin working on profile, title                             |
|     The next step is to make sure that your résumé is as up-to-date   as possible.                                                                                                                                    |     L’étape suivante   consiste à vous assurer que votre CV est aussi à jour que possible.                                                                                                                                                                                       |     Application step to begin working on profile, description,   paragraph one      |
|     More specifically, not only do we want to know your work   history, but we highly value knowing about your community experience, awards,   personal initiatives, and education too!                               |     Plus précisément, non   seulement nous voulons connaître votre parcours professionnel, mais également   votre expérience communautaire, vos récompenses, vos initiatives personnelles   et votre éducation!                                                                  |     Application step to begin working on profile, description,   paragraph two      |
|     Once you’ve completed your resume and are happy with the   experiences you’ve added, you’ll use them in further steps to help us better   understand how you meet the skill requirements for this opportunity.    |     Une fois que vous avez   mis votre CV à jour et que vous êtes satisfait des expériences que vous avez   ajoutées, vous les utiliserez dans d’autres étapes pour nous aider à mieux   comprendre comment vous répondez aux exigences de compétences pour cette   occasion.    |     Application step to begin working on profile, description,   paragraph three    |


## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Page exists under the appropriate url
- [ ] Page is accessible only when the feature flag "Application Revamp" is ON 
- [ ] Hero subtitle and breadcrumbs are correct
- [ ] The inner contents of the page (so not the hero and stepper layout) have a storybook story
- [ ] Save and quit for now is just a link back to Application dashboard
- [ ] Let's go button does not save anything, its just a link to Resume step


## :stop_sign: Blockers
Blocked by #6027

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #6039 

## 👋 Introduction

Update the intro to resume review page. 

## 🕵️ Details

Translations remaining
```
  "AOrJqm": {
    "defaultMessage": "Got it, let's go!",
    "description": "Link text to continue the application process"
  },
```
Will pluck save and quit off Eric's PR

## 🧪 Testing

1. Create a draft application, update with  `"WELCOME", "REVIEW_YOUR_PROFILE"`
2. Head to `/en/applications/:id/resume/introduction`
3. Observe page
4. Run Storybook and ensure no errors

## 📸 Screenshot

image

### CODE REVIEW

**Review by esizer — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/Applications/ApplicationResumeIntroductionPage/ApplicationResumeIntroductionPage.tsx [hunk 1]
```diff
@@ -47,14 +48,80 @@ export const getPageInfo: GetApplicationPageInfo = ({
...
+            "The next step is to make sure that your résumé is as up-to-date as possible.",
+          id: "kfXCY8",
+          description:
+            "Application step to begin working on résumé, paragraph one",
+        })}
+      </p>
+      <p data-h2-margin="base(x1, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            "More specifically, not only do we want to know your work history, but we highly value knowing about your community experience, awards, personal initiatives, and education too!",
```
[the remark this answers is not in the document]
Comment (esizer) [reply apps/web/src/pages/Applications/ApplicationResumeIntroductionPage/ApplicationResumeIntroductionPage.tsx:75]: It is highly subjective so it is hard to give a good answer. For this use case where we are using it to draw attention to specific terms, i'd say it is useful here 👍

[the remark this answers is not in the document]
Comment (mnigh) [reply apps/web/src/pages/Applications/ApplicationResumeIntroductionPage/ApplicationResumeIntroductionPage.tsx:75]: Plus précisément, non seulement nous voulons connaître votre **parcours professionnel**, mais également votre **expérience communautaire**, vos **récompenses**, vos **initiatives personnelles** et votre **éducation**!

[the remark this answers is not in the document]
Comment (vd1992) [reply apps/web/src/pages/Applications/ApplicationResumeIntroductionPage/ApplicationResumeIntroductionPage.tsx:75]: Thanks!
🇫🇷

**Review by esizer — APPROVED** [review]
LGTM! 💪 Hopefully that translation comes in 🤞

**Comment by vd1992** [comment]
> LGTM! 💪 Hopefully that translation comes in 🤞

Got 'em!

image

image

**Pull request merged**